tools/depends/native/gen_entitlements-native/ios11_entitlements.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this is target not native

Reply:
A target file of course. Not sure if I share the separation you are after here ;) Imo the signing process runs on native so it’s native ;)

Reply:
if this xml file is dependent on target, i.e. differs between IOS versions, it should be a target